sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -68,9 +71,44 @@ case class ScriptTransformation(
       val errorStream = proc.getErrorStream
       val reader = new BufferedReader(new InputStreamReader(inputStream))
 
-      val (outputSerde, outputSoi) = ioschema.initOutputSerDe(output)
+      // TODO make the 2048 configurable?
```

Comment:
I don't mind cleaning up these comments; it might also be nice to edit some of the other comments to refer to the relevant JIRAs, since the code movement here makes it a bit harder to find them using `git blame`.